homeassistant/components/jewish_calendar/sensor.py

```diff
@@ -3,16 +3,23 @@
 
 import voluptuous as vol
 
+import datetime as dt
+
 from homeassistant.components.sensor import PLATFORM_SCHEMA
 from homeassistant.const import (
-    CONF_LATITUDE, CONF_LONGITUDE, CONF_NAME, SUN_EVENT_SUNSET)
+    CONF_LATITUDE, CONF_LONGITUDE, CONF_NAME, SUN_EVENT_SUNSET, CONF_SCAN_INTERVAL)
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.entity import Entity
 from homeassistant.helpers.sun import get_astral_event_date
 import homeassistant.util.dt as dt_util
+from homeassistant.helpers.event import async_track_sunset, async_call_later, async_track_time_change
+
+import hdate
```

Comment:
Is this ok now? I was under the impression that external libraries should only be imported during the actual usage.

https://developers.home-assistant.io/docs/en/0.85/creating_component_deps_and_reqs.html

Reply:
I missed that note.

Since we need hdate in the setup too, it didn't made much sense for me to import it twice, but yes we can change that.